website/docs/r/resource_group.html.markdown

```diff
@@ -74,17 +76,18 @@ The following arguments are supported:
   part of the resource group. Groups can be nested up to 3 levels deep and can be combined by 
   individual filters. See the [api-docs](https://lwdocs-rg2.netlify.app/api/api-resource-group/#filterable-fields) for the supported fields.
   Each `group` must have at least one of `group` or `filter` defined.
-* `type` - (Required) The type of resource group being created. e.g. AWS/GCP/AZURE
+* `type` - (Required) The type of resource group being created, AWS, GCP, or AZURE
 * `description` - (Optional) The description of the resource group.
 * `enabled` - (Optional) The state of the external integration. Defaults to `true`.
 
 ## Import
 
-A Lacework Resource Group can be imported using a `RESOURCE_GROUP_GUID`, e.g.
+You can import a Lacework resource group by `RESOURCE_GROUP_GUID`, for example:
 
 ```
 $ terraform import lacework_resource_group.example EXAMPLE_1234BAE1E42182964D23973F44CFEA3C4AB63B99E9A1EC5
 ```
--> **Note:** To retrieve the `RESOURCE_GROUP_GUID` from existing resource groups in your account, 
+
+**Note:** To retrieve the `RESOURCE_GROUP_GUID` from existing resource groups in your account, 
```

Comment:
Let's revert this change since the Sigil `->` will render a box on the docs. Like this one https://registry.terraform.io/providers/lacework/lacework/latest/docs/resources/integration_gcr#credentials

![Screenshot 2023-08-30 at 10 05 46 AM](https://github.com/lacework/terraform-provider-lacework/assets/5712253/683510ff-640a-47fb-b006-6c654eb06334)

More information about Terraform provider documentation can be found here:
https://developer.hashicorp.com/terraform/registry/providers/docs#callouts